test/complex.jl

```diff
@@ -1,6 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
 using LinearAlgebra
+using Test
```

Comment:
```suggestion
```
We don't actually need a `using Test` here. Because of the way these files are included we can assume that `Test` is loaded beforehand.